ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/recovery/RecoveryMonitorTest2.java

```diff
@@ -109,7 +108,8 @@ public void start(Xid xid, int i) throws XAException {
 
         RecoveryManager manager = RecoveryManager.manager(RecoveryManager.DIRECT_MANAGEMENT);
 
-        manager.addModule(new XARecoveryModule()); // we only need to test the XARecoveryModule
```

Comment:
Same comment here (the change is unrelated to the issue)

Reply:
same comment as above (changes are related to this issue)

Reply:
Thanks

Reply:
Apologies for not checking. Thanks